siac/parse.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"math/big"
+	"strings"
+)
+
+var ErrUnableToParseSize = errors.New("unable to parse size")
```

Comment:
we prefer lower-case (project is inconsistent about this because we only started enforcing the convention recently).

`errUnableToParseSize`.

Our general approach is:

1. return errors.New
if we need to check for a specific error (in testing or otherwise):
2. return errUnexportedErr (no docstring needed on error)
if external packages need to check for a specific error
3. return ErrExportedErr (docstring needed on error)